DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -1252,6 +1355,14 @@ defaultproperties
     bingo_options(103)=(event="ManWhoWasThursday",desc="Read 4 parts of The Man Who Was Thursday",max=4)
     bingo_options(104)=(event="GreeneArticles",desc="Read 4 newspaper articles by Joe Greene",max=4)
     bingo_options(105)=(event="MoonBaseNews",desc="Read news about the Lunar Mining Complex",max=1)
+    bingo_options(106)=(event="06_Datacube_05",desc="Learn Maggie Chow's Birthday",max=1)
+    bingo_options(107)=(event="Gray_ClassDead",desc="Kill 5 Grays",max=5)
+    bingo_options(108)=(event="CloneCubes",desc="Read about the four clones in Area 51",max=4)
```

Comment:
Reading about the clones is really funny in Randomizer lol, these datacubes are gonna be all over the place